src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -59,35 +52,29 @@ public function showAction(Request $request)
 
     /**
      * @param Request $request
-     * @param string  $namespace
      *
      * @return Response
      */
-    public function updateAction(Request $request, $namespace)
+    public function updateAction(Request $request)
     {
-        $this->isGrantedOr403($namespace);
+        $schemaAlias = $request->get('schema');
 
-        $manager = $this->getSettingsManager();
+        $this->isGrantedOr403($schemaAlias);
```

Comment:
I am not sure about the naming, ``namespace`` was not perfect, but the goal was to separate ``schema`` (how things are structured) from namespace (where they are saved). I know we have resolvers, but ``load(schema)`` can be confusing. SchemaResolvers make sense - absolutely. Perhaps we could come up with something more clear for the loading - ``type`` of settings? Not sure.